package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sevakitchen",
+  "name": "proso-io",
```

Comment:
why project name change?

Reply:
sorry because ive named the directory as proso-io in my local computer...will change that to sevakitchen

Reply:
is it not changed yet?